tools/mmp/driver.cs

```diff
@@ -800,6 +808,14 @@ static void Pack (IList<string> unprocessed)
 			if (App.LinkMode != LinkMode.All && App.RuntimeOptions != null)
 				App.RuntimeOptions.Write (App.AppDirectory);
 
+			if (aot.IsAOT) {
+				if (!IsUnified || !Is64Bit)
+					throw new MonoMacException (98, true, "AOT compilation is only available on Unified 64-bit", ret);
```

Comment:
Waiting to hear back from @kumpera if safe to use mono64 we ship internally for 32-bit before flipping back on.